inc/spbc-settings.php

```diff
@@ -2215,10 +2224,8 @@ function spbc_field_traffic_control_log()
 
         // Subscription should be renewed
     } elseif ( ! $spbc->moderate) {
-        $button = '<input type="button" class="button button-primary" value="' . __('RENEW', 'security-malware-firewall') . '"  />';
-        $link   = sprintf('<a target="_blank" href="https://cleantalk.org/my/bill/security?cp_mode=security&utm_source=wp-backend&utm_medium=cpc&utm_campaign=WP%%20backend%%20trial_security&user_token=%s">%s</a>', $spbc->user_token, $button);
         echo '<div style="margin-top: 10px;">'
-             . '<h3 style="margin: 5px; display: inline-block;">' . __('Please renew your security license.', 'security-malware-firewall') . '</h3>' . $link .
+             . '<h3 style="margin: 5px; display: inline-block;">' . __('Please give us a few seconds to pull account data from the cloud.', 'security-malware-firewall') . '</h3>' .
```

Comment:
This is the banner that shown if the license expire. Please, check if the correct place selected for this new message.

Reply:
Thanks, fixed https://github.com/CleanTalk/security-malware-firewall/pull/293/commits/4e108b8d67e691e7abde852afc8b2dd0d003f72b